src/Host.h

```diff
@@ -178,16 +178,16 @@ class Host : public QObject
 
     bool mPrintCommand;
 
-    // The following was incorrectly called mRawStreamDump
-    // and caused the log file to be in HTML format rather
-    // then plain text.  To cover the corner case of the user
-    // changing the mode whilst a log is being written it has
-    // been split into:
-    bool mIsNextLogFileInHtmlFormat;
-
-    // What the user has set as their preference
+    // To cover the corner case of the user changing the mode
+    // whilst a log is being written, this stores the mode of
+    // the current log file. See mIsNextLogFileInHtmlFormat.
     bool mIsCurrentLogFileInHtmlFormat;
```

Comment:
Suggested revised text:

    // To cover the corner case of the user changing the mode
    // whilst a log is being written, this stores the mode of
    // the current log file and is set from
    // mIsNextLogFileInHtmlFormat at the point that a log is started.